### PR TITLE
[RITP] auto things and shooter map

### PR DESCRIPTION
**Pull request by WindowsVistaisCool**
I trust most path changes are fine. If shooter map works then great

## Good autos:
- `3-SF-[C2-C3]-EC`

## Autos that still need logic work:
- `3-SB-[F4-F3]-EF`: SCollect -> SCollectTele, CandSub -> SmartShoot
- `2-SB-[F4]-EF`: Could easily be turned into a 3 piece; needs more testing.
- `2-ST-[C1]-EC-CAND`: probably should be deleted, but will leave for now

### CODE REVIEW

**Review by Meh243 — COMMENTED** [review]
No problems that I have saw. Nice fix for teleop!

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -272,7 +272,7 @@ protected void configureButtonBindings() {
 				.whileTrue(new PointBlankShot(flywheel, pivot).deadlineWith(leds.enableState(LED_STATES.SHOOTING)));
 		// new Trigger(coPilot::getYButton).whileTrue(new PivotUP(pivot));
 		// new Trigger(coPilot::getYButton).whileTrue(new NotePass(drivetrain, flywheel, pivot, driver, indexer));
-		// new Trigger(coPilot::getYButton).whileTrue(new Tune(flywheel, pivot));
+		new Trigger(coPilot::getYButton).whileTrue(new Tune(flywheel, pivot));
```
Comment (HeeistHo) [inline src/main/java/frc/robot/RobotContainer.java:275]: comment this

File: src/main/java/frc/robot/command/SmartCollect.java [hunk 2]
```diff
@@ -39,23 +39,30 @@ public class SmartCollect extends Command {
...
+            Indexer indexer, Pivot pivot, Flywheel flywheel, boolean isAutonomous) {
         this.collector = collector;
         this.indexer = indexer;
         this.pivot = pivot;
         this.flywheel = flywheel;
         this.collectorPower = collectorPower;
         this.indexerPower = indexerPower;
 
-        if (DriverStation.isAutonomous()) {
+        if (isAutonomous || DriverStation.isAutonomous()) {
```
Comment (HeeistHo) [inline src/main/java/frc/robot/command/SmartCollect.java:53]: why is this needed?

**Review by HeeistHo — CHANGES_REQUESTED** [review]
comment tune.

**Review by HeeistHo — APPROVED** [review]
yay

**Pull request merged**